examples/serviceenablement/serviceenablement.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/services/serviceenablement"
+	"github.com/stackitcloud/stackit-sdk-go/services/serviceenablement/wait"
+)
+
+func main() {
+	// Specify the project ID
+	projectId := "PROJECT_ID"
+
+	// Create a new API client, that uses default authentication and configuration
+	client, err := serviceenablement.NewAPIClient(
+		config.WithRegion("eu01"),
+	)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Creating API client: %v\n", err)
+		os.Exit(1)
+	}
+
+	// List Services
+	listServicesResp, err := client.ListServiceStatusExecute(context.Background(), projectId)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error when calling `ListServices`: %v\n", err)
+	} else {
+		fmt.Printf("Number of services: %v\n", len(*listServicesResp.Items))
+	}
+
+	// Get Service Status
+	serviceId := "SERVICE_ID"
```

Comment:
In the example we could get the `serviceId` from the response of the previous call, right?

Reply:
Yes we can, doing that